tests/unit/certtool.py

```diff
@@ -0,0 +1,100 @@
+import datetime
```

Comment:
Since this class is only used in the inline cert test, it might be an option to move it to `tests/unit/test_db_config_options.py` instead of it having its own file.

Reply:
Refactored...